src/app/migration/src/src/app/modules/workspace/components/content-creation-forms/create-collection/create-collection.component.ts

```diff
@@ -93,11 +98,13 @@ export class CreateCollectionComponent implements OnInit {
          this.userProfile = user.userProfile;
         }
       });
-    this.showLoader = false;
     this.userForm = this.formBuilder.group({
       name: '',
       description: ''
     });
+    if (document.getElementById('collectionEditor')) {
```

Comment:
why we need to remove 'collectionEditor', which is present in a different route. Which should be destroyed when route changes